intf/vernacexpr.mli

```diff
@@ -450,8 +445,8 @@ type vernac_expr =
   | VernacRegister of lident * register_kind
   | VernacComments of comment list
 
-  (* Stm backdoor *)
-  | VernacStm of vernac_expr stm_vernac
+  (* Stm backdoor: deprecated, to be removed *)
```

Comment:
As for the previous comment, I don't think the comment is appropriate.  It should be
`Rewrite fake_id to collect feedback about finished jobs, then remove this.`